### PR TITLE
Explicitly state that the vite.config.ts file needs to be created 

### LINKED ISSUE #795: [docs] Vite Quickstart is Misleading about vite.config.ts
**Article in question**
https://tauri.app/v1/guides/getting-started/setup/vite/

**Part in question**
> Before we move on, let's quickly customize our `vite.config.ts` file, so you get the best compatibility with Tauri.

**Issue**
No `vite.config.ts` file is created by following the instructions.

_From searching through the Discord I gathered this file has to be manually created at the project root instead when following the described process._

### PR DESCRIPTION
**Pull request by scubamaggo**
Attempts to resolve #795

### CODE REVIEW

**Comment by FabianLars** [comment]
This is already quite good, thanks!
But this is only true for the vanilla/vanilla-ts templates, all the other ones to come with a default file for calling the respecting framework's plugin, basically just this line: <https://github.com/FabianLars/mw-toolbox/blob/main/gui/vite.config.ts#L10>

Do you have an idea how we could integrate that into the text nicely?

File: docs/guides/getting-started/setup/vite.mdx [hunk 1]
```diff
@@ -62,7 +62,7 @@ pnpm create vite
 3. **Select a variant**  
    Vite offers [TypeScript] and vanilla JavaScript variants for all templates, and you can select the variant here. We **strongly** recommend TypeScript as it helps you write more secure, maintainable code faster and more efficiently. For this guide, we assume you choose `vanilla-ts`.
 
-Before we move on, let's quickly customize our `vite.config.ts` file, so you get the best compatibility with Tauri.
+When starting the frontend via the `vite` command, Vite will look for a config file named `vite.config.ts` inside the project root. We want to customize this file to get the best compatibility with Tauri. Since it is not created by the scaffolding above, create the `vite.config.ts` file in the root directory of the project and add the following content:
```
Comment (lorenzolewis) [inline docs/guides/getting-started/setup/vite.mdx]: Could try something like this
```suggestion
When starting the frontend via the `vite` command, Vite will look for a config file named `vite.config.ts` inside the project root. We want to customize this file to get the best compatibility with Tauri. If it is not created by the scaffolding above (such as if you're using vanilla JavaScript), you may need to create the `vite.config.ts` file in the root directory of the project. Update the file with the following content: 
```

Comment (FabianLars) [reply docs/guides/getting-started/setup/vite.mdx]: Sounds good to me 👍

File: docs/guides/getting-started/setup/vite.mdx [hunk 2]
```diff
@@ -62,7 +62,7 @@ pnpm create vite
 3. **Select a variant**  
    Vite offers [TypeScript] and vanilla JavaScript variants for all templates, and you can select the variant here. We **strongly** recommend TypeScript as it helps you write more secure, maintainable code faster and more efficiently. For this guide, we assume you choose `vanilla-ts`.
 
-Before we move on, let's quickly customize our `vite.config.ts` file, so you get the best compatibility with Tauri.
+When starting the frontend via the `vite` command, Vite will look for a config file named `vite.config.ts` inside the project root. We want to customize this file to get the best compatibility with Tauri. If it is not created by the scaffolding above (such as if you're using vanilla JavaScript), you may need to create the `vite.config.ts` file in the root directory of the project. Update the file with the following content:
```
Comment (lorenzolewis) [inline docs/guides/getting-started/setup/vite.mdx]:
```suggestion
When starting the frontend via the `vite` command, Vite will look for a config file named `vite.config.ts` inside the project root. We want to customize this file to get the best compatibility with Tauri. If it is not created by the scaffolding above (such as if you're using vanilla JavaScript), you may need to create the `vite.config.ts` file in the root directory of the project.

Update the file with the following content: 
```

**Review by lorenzolewis — APPROVED** [review]

**Pull request merged**